scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -631,10 +634,58 @@ def get_uri(self, token_expiration: timedelta = timedelta(hours=1)) -> str:
             self._token = ""
             return uri
 
+    def get_container_client(self) -> ContainerClient:
+        """Get container client.
+
+        Returns:
+            ContainerClient: Container client object for the asset.
+        """
+        # Get URL to container, while preserving any SAS token
+        model_uri = urllib.parse.urlparse(self.uri)

```

Comment:
`self._uri`?

Reply:
same for the container_name, either container_name or self._container_name

Reply:
I'll look at the uses of `container_name`, but in this case I really want to use the cached uri if available, so `self.uri` is intentional.